bokehjs/src/lib/models/transforms/jitter.ts

```diff
@@ -35,10 +36,28 @@ export class Jitter extends RangeTransform {
   }
 
   override v_compute(xs0: Arrayable<number | Factor>): Arrayable<number> {
-    if (this.previous_values != null && this.previous_values.length == xs0.length)
-      return this.previous_values
-    this.previous_values = super.v_compute(xs0)
-    return this.previous_values
+    let xs: Arrayable<number>
+    if (super.range instanceof FactorRange)
+      xs = super.range.v_synthetic(xs0)
+    else if (isArrayableOf(xs0, isNumber))
+      xs = xs0
+    else
+      throw new Error("unexpected")
```

Comment:
This would be better replaced with `unreachable()` from `core/util/assert`. The code in the parent class wasn't updated when this utility function was added.

Reply:
Changed to using `unreachable()`

Reply:
Also changed in other files that is throwing the same error